src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -64,7 +66,17 @@ protected function executeCommand(array $command_args)
         if ($this->validateCommand($command_args)) {
             $command_line = $this->getCommandLine($command_args);
 
-            $result = $this->environment->sendCommandViaSsh($command_line);
+            $logger = $this->log();
+            $result = $this->environment->sendCommandViaSsh(
+                $command_line,
+                function ($type, $buffer) use ($logger) {
+                    if (Process::ERR === $type) {
+                        $logger->error($buffer);
+                    } else {
+                        $logger->notice($buffer);
```

Comment:
@TeslaDethray just recapping our discussion. I think we agreed that the command output (`$buffer`) should be sent to the outputter ... or STDOUT if that is a more reasonable short-term fix.